packages/tokens/bin/lib/formats/formatSvg.ts

```diff
@@ -11,16 +11,27 @@ const cleanSvgMarkup = (name: string, svgMarkup: string): string => {
         plugins: [
             {name: 'removeViewBox', active: false},
             {name: 'removeDimensions', active: true},
-            {
-                name: 'removeAttributesBySelector',
-                active: true,
-                params: {selector: '[stroke]', attributes: ['stroke']} as any,
-            },
             {
                 name: 'addAttributesToSVGElement',
                 active: true,
-                params: {attributes: [{stroke: 'currentColor'}, {width: '1em'}, {height: '1em'}]},
+                params: {attributes: [{width: '1em'}, {height: '1em'}]},
             },
+            {
+                type: 'visitor',
```

Comment:
visitor? :thinking:

Reply:
its a type of svgo plugin that "visits" every html element one by one

Reply:
oh I see :eyes: Interesting :+1: